test/e2e/specs/signing.spec.ts

```diff
@@ -84,13 +84,13 @@ describe('Signing ane exporting test', () => {
 			await tapBack();
 			await tapBack();
 			await testTap(SecurityHeader.scanButton);
-			await testVisible(SignedTx.qrView);
+			await testExist(SignedTx.qrView);
 		});
 
-		it('should sign transfer request', async () => {
-			await launchWithScanRequest(ScanTestRequest.TransferExtrinsicKusama);
-			await testSignedTx();
-		});
+		// it('should sign transfer request', async () => {
+		// 	await launchWithScanRequest(ScanTestRequest.TransferExtrinsicKusama);
+		// 	await testSignedTx();
+		// });
```

Comment:
What is going on here? Should it be removed?

Reply:
This is the one test which in the CI was causing a setState issue and would fail. That being said when running the test locally there is no issues, or errors. So for now I have it commented out. But am actively trying to figure it out. It is just incredibly hard to debug when the only time its erroring is in the CI.

Reply:
Would be good to log an issue and add a ref to it here as a comment.

Reply:
This is fixed below now